autoendpoint/src/extractors/subscription.rs

```diff
@@ -111,7 +111,9 @@ impl FromRequest for Subscription {
                     .unwrap_or_default();
                 // For now, record that we had a good (?) VAPID sub,
                 app_state.metrics.incr("notification.auth.ok")?;
-                info!("VAPID sub: {:?}", sub)
+                if sub.is_some() {
+                    info!("VAPID sub: {:?}", sub);
+                }
```

Comment:
Nit probably instead?
```suggestion
                if let Some(sub) = sub {
                    info!("VAPID sub: {sub}");
                }
```